app/stores/MessageUnreadStore.js

```diff
@@ -90,8 +99,10 @@ let UnreadStore = assign({}, BaseStore, {
                     }
 
                     _unreadBool = _unreadBool.set(backEndChannelId, _unread[backEndChannelId].latestMessageId > _unread[backEndChannelId].lastSeenMessageId);
+                    getMessagesCount(backEndChannelId, {id:_unread[backEndChannelId].lastSeenMessageId?_unread[backEndChannelId].lastSeenMessageId:0});
```

Comment:
I modify a little to avoid undefined problem of lastSeenMessageId, @WangBoxue  you can check it, if the user never check "team_1", and he come to team_1 first time, this bug happened.